test/token/ERC20/extensions/ERC4626.test.js

```diff
@@ -21,6 +22,30 @@ contract('ERC4626', function (accounts) {
     }
   });
 
+  it('asset has not yet been created', async function () {
+    for (let i = 0; i < 6; i++) {
+      const vault = await ERC4626.new('', '', accounts[i]);
```

Comment:
why do we need to check for multiple EOA. Isn't one enough ?

Reply:
since `accounts` is available I thought we can do a simple "EOA" fuzzing test, but generally, yes one should be enough. Lmk your preferences.

Reply:
Since we're already using the [ERC4626 test suite](https://github.com/a16z/erc4626-tests/blob/8b1d7c2ac248c33c3506b1bff8321758943c5e11/ERC4626.test.sol) with Foundry, we can add that fuzzing case for the accounts.

Reply:
I had to force myself this morning to write `JS` tests lol - I actually wanted to first just simply open a PR with the `forge` tests but I thought you will reject that in any case :). Happy to add some fuzz tests if you guys agree on this path.

Reply:
I think [this reference ](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/7b3e7b7055d6e1226d4920ca7d7c446c0c0bfff5/test/token/ERC20/extensions/ERC4626.t.sol) might help you. It'd be a matter of extending `ERC4626StdTest`

> Happy to add some fuzz tests if you guys agree on this path.

We'd appreciate those tests if they make sense. At least this one does for me.

Reply:
Thx, am actually familiar with it since I used it for my Vyper `ERC4626` tests [here](https://github.com/pcaversaccio/snekmate/blob/main/test/extensions/ERC4626.t.sol) as well (yes I use Solidity tests to test Vyper :)).

Reply:
Actually writing my own tests has led to opening an issue here (I was really paranoid about everything tbh).

Reply:
@Amxx I refactored it into using only one EOA which seems fine.